icon-registry/icons/os-generic_x16.svg

```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_4669_8471)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 8C2 4.68629 4.68629 2 8 2C11.3137 2 14 4.68629 14 8C14 11.3137 11.3137 14 8 14C4.68629 14 2 11.3137 2 8ZM8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0ZM7.97363 8.00439C7.97363 9.62158 7.00684 10.5312 5.42041 10.5312C3.82959 10.5312 2.87158 9.61719 2.87158 8.00439V7.99561C2.87158 6.41357 3.86475 5.46875 5.42041 5.46875C6.98047 5.46875 7.97363 6.40039 7.97363 7.99561V8.00439ZM4.69971 8.00439C4.69971 8.77344 4.96777 9.19531 5.4248 9.19531C5.87305 9.19531 6.14551 8.76904 6.14551 8.00439V7.99561C6.14551 7.24414 5.85986 6.80469 5.42041 6.80469C4.97656 6.80469 4.69971 7.24414 4.69971 7.99561V8.00439ZM13.1284 8.83496C13.1284 9.88086 12.188 10.5312 10.7817 10.5312C9.25244 10.5312 8.45264 9.87646 8.34277 8.90967L8.33838 8.87012H10.0786L10.0874 8.90527C10.1577 9.18652 10.3774 9.35352 10.7817 9.35352C11.1289 9.35352 11.3442 9.23486 11.3442 9.0415V9.03711C11.3442 8.86133 11.1992 8.76025 10.7993 8.68555L9.92041 8.52295C8.95801 8.34717 8.44824 7.83301 8.44824 7.08154V7.07715C8.44824 6.0708 9.30518 5.46875 10.7158 5.46875C12.2188 5.46875 12.9834 6.21582 12.9834 7.09912V7.12988H11.3486L11.3442 7.09033C11.3179 6.86182 11.1025 6.64648 10.7158 6.64648C10.3862 6.64648 10.1709 6.76953 10.1709 6.97607V6.98047C10.1709 7.15186 10.2939 7.24854 10.7334 7.33203L11.6123 7.49463C12.6582 7.68799 13.1284 8.09668 13.1284 8.83057V8.83496Z" fill="currentColor"/>
+</g>
+<defs>
+<clipPath id="clip0_4669_8471">
+<rect width="16" height="16" fill="currentColor"/>
```

Comment:
this is the only one where it makes sense to have the classes. But do you really need this con for your dev ?

Reply:
Yes I need it when there are multiple groups for a test. I would display this icon and the number of operating systems in the groups e.g. `<os-generic/> 2 operating systems`